refresh/fromswagger/swaggerize.js

```diff
@@ -196,8 +185,11 @@ exports.parse = function (memfs, swaggerPath) {
   debug('in parse')
   return loadAsync(memfs, swaggerPath)
     .then(loaded => {
-      ensureValid(loaded, swaggerPath)
-      debug('successfully validated against schema')
-      return { loaded: loaded, parsed: parseSwagger(loaded) }
+      var loadedAsJSONString = JSON.stringify(loaded)
+      return ensureValid(loaded, swaggerPath)
```

Comment:
It would be nice to preserver the debug log message for when validation is successful.